src/main/java/com/deavensoft/paketomat/center/PackageServiceImpl.java

```diff
@@ -19,57 +19,53 @@
 
 @Slf4j
 @Service
-public class CenterServiceImpl implements CenterService {
-    private final CenterRepository centerRepository;
+@AllArgsConstructor
+public class PackageServiceImpl implements PackageService {
+    private final PackageRepository packageRepository;
     private final EmailService emailService;
     @Autowired
-    public CenterServiceImpl(@Qualifier("center") CenterRepository centerRepository,EmailService emailService) {
-        this.centerRepository = centerRepository;
-        this.emailService=emailService;
-    }
-
 
     public List<Package> getAllPackages() {
-        return centerRepository.findAll();
+        return packageRepository.findAll();
     }
 
     public void save(Package p) {
-        centerRepository.save(p);
+        packageRepository.save(p);
     }
 
     public Optional<Package> findPackageById(Long id) {
-        return centerRepository.findById(id);
+        return packageRepository.findById(id);
     }
 
     public void deletePackageById(Long id) {
-        centerRepository.deleteById(id);
+        packageRepository.deleteById(id);
     }
 
     @Override
     public void deleteAll() {
-        centerRepository.deleteAll();
+        packageRepository.deleteAll();
     }
 
     @Override
     public void updateStatus(Long code, Status status) {
-        List<Package> packages = centerRepository.findAll();
+        List<Package> packages = packageRepository.findAll();
         for (Package p : packages) {
             if(p.getCode().equals(code)){
```

Comment:
You can directly use `packageRepository.findByCode()`

Reply:
Done.